#This message is used to send command to Herkulex
# Tilt module Angle(degree) Move Command

float64 target_angle
int16 playtime
---
bool command_result
